src/components/Gallery/hooks/useImageZoom/constants.ts

```diff
@@ -0,0 +1,7 @@
+// Constants
+export const MIN_SCALE = 1;
+export const MAX_SCALE_DESKTOP = 2; // Desktop max zoom
+export const MAX_SCALE_TOUCH = 3; // Mobile max zoom
+export const DRAG_THRESHOLD = 3; // px - minimum movement to consider as drag
+export const DOUBLE_TAP_DELAY = 300; // ms
```

Comment:
Using _MS suffix will describe, that this is the milliseconds
The same for DOUBLE_TAP_DISTANCE and DRAG_THRESHOLD